docs/1.1-DRAFT/workflows.md

```diff
@@ -169,26 +175,70 @@ A workflow diagram may still be provided even if there is no programmatic `Softw
 
 ## Complying with BioSchemas Computational Workflow profile
 
-To comply with the BioSchemas [ComputationalWorkflow profile],
-where possible, data entities representing _workflows_ SHOULD describe these properties and their related contextual entities:
+Data entities representing _workflows_ (`@type: ComputationalWorkflow`)
+SHOULD comply with the
+BioSchemas [ComputationalWorkflow profile],
+where possible. When complying with this profile, the workflow data entities
+MUST describe these properties and their related contextual entities:
 
 * [name] giving a short descriptive human-readable name for the workflow
 * [programmingLanguage] identifying the workflow system, typed as `ProgrammingLanguage`
 * [creator] identifying the [Person](s) and/or [Organization](s) that assembled the workflow. (This may differ from the [author] of the conceptual workflow, e.g. if the workflow was converted from one language to another).
 * [dateCreated] the date the workflow was first created, e.g. `2020-05-23`
 * [license] identifying a [CreativeWork] that details the license for distributing or editing the workflow
 * [input] and [output] identifying contextual entities for the [FormalParameter] describing input and output parameters/variables that may be varied on different workflow executions
-* [sdPublisher] to identify the [Person] or [Organization] who has made the JSON-LD description of the workflow
-* [url] to identify a public webpage or source code repository for the workflow
+* [sdPublisher] to identify the [Person] or [Organization] who has made this JSON-LD description of the workflow
+* [url] to identify a public page or source code repository for the workflow
```

Comment:
Schema.org has this awkward split between https://schema.org/SoftwareApplication and https://schema.org/SoftwareSourceCode but many useful properties are only on `SoftwareApplication` or even `WebAPI`.

Perhaps Bioschemas can propose that https://schema.org/documentation also applies to `SoftwareApplication` and `SoftwareSourceCode` rather than just https://schema.org/WebAPI - it would also make sense for `documentation` to be subproperty of https://schema.org/subjectOf  (it might also document other things)

There are many potential http://schema.org/CreativeWork fields like:

* https://schema.org/url - the homepage of the workflow itself <https://nf-co.re/chipseq>
* https://schema.org/identifier - a persistent identifier for workflow e.g. DOI <https://zenodo.org/record/3966161>
* https://schema.org/citation - a scholarly article that (ideally) talks about this workflow e.g. <https://doi.org/10.1038/s41587-020-0439-x.>
* https://schema.org/mainEntityOfPage (referencing a page that is mainly about this workflow - e.g. documentation <https://biobb.readthedocs.io/> or registration <https://workflowhub.eu/workflows/56>
* https://schema.org/discussionUrl (a place to discuss this workflow - e.g. <https://ask.bioexcel.eu/c/BioExcel-Building-Blocks-library/23> or even GitHub
* https://schema.org/subjectOf - documentation page that talks about this workflow, but also other things <http://mmb.irbbarcelona.org/biobb/workflows>
* https://schema.org/isBasedOn - another workflow/pipeline it is based on, e.g. <http://www.mdtutorials.com/gmx/lysozyme/index.html>
* https://schema.org/usageInfo - tutorials e.g. <http://mmb.irbbarcelona.org/biobb/workflows/tutorials/md_setup>
* https://schema.org/workExample - more specific example that realize this workflow e.g. <https://github.com/bioexcel/pmxlaunchCV19/blob/master/workflows/CV19/pmx_cv.py> realizes the tutorial workflow by adding specific data

And from SoftwareSourceCode:
* https://schema.org/codeRepository - GitHub repo (root?) e.g. <https://github.com/nf-core/chipseq>

Example of needing several of these fields in WorkflowHub for the BioBB workflow, under _Discussion channels_ on the right of https://workflowhub.eu/workflows/56:

![discussion-channels](https://user-images.githubusercontent.com/253413/94792761-5be95a80-03d1-11eb-9971-83e69b1d1983.png)

@dgarijo if we want to add how to link to documentation, we would need to add it in the section above bioschemas conformance, as [Bioschemas ComputationalWorkflow](https://bioschemas.org/profiles/ComputationalWorkflow/0.5-DRAFT-2020_07_21/) does not have a view on this yet. Do you think we add this now, or we wait for next release of ComputationalWorkflow profile?

@alaninmcr  shall we feed this back into the next BioSchemas profile on how to link to workflow documentation? Make issue(s) in https://github.com/BioSchemas/specifications

Reply:
Yes a lot of these seem to be suggestions for the ComputationalWorkflow profile, rather than about RO-Crate. Is there a better way of separating the two?